libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -815,6 +827,20 @@ void test_MQTT_Connect_happy_path()
     status = MQTT_Connect( &mqttContext, &connectInfo, &willInfo, timeout, &sessionPresent );
     TEST_ASSERT_EQUAL_INT( MQTTSuccess, status );
     TEST_ASSERT_EQUAL_INT( MQTTConnected, mqttContext.connectStatus );
+
+    /* Request to establish a session if present and session present is received
+     * from broker. */
+    mqttContext.connectStatus = MQTTNotConnected;
+    connectInfo.cleanSession = false;
+    sessionPresent = true;
+    MQTT_GetIncomingPacketTypeAndLength_ExpectAnyArgsAndReturn( MQTTSuccess );
+    MQTT_GetIncomingPacketTypeAndLength_ReturnThruPtr_pIncomingPacket( &incomingPacket );
+    MQTT_DeserializeAck_ExpectAnyArgsAndReturn( MQTTSuccess );
+    MQTT_DeserializeAck_ReturnThruPtr_pSessionPresent( &sessionPresent );
+    status = MQTT_Connect( &mqttContext, &connectInfo, NULL, timeout, &sessionPresentResult );
```

Comment:
The other tests pass `sessionPresent` to MQTT_Connect. I'd swap these two around and possibly change the name of the return pointer to `sessionPresentExpected` or something to be consistent with the other tests

Reply:
ok.